rainbowup/scripts/prepare.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This script does not work when location of nearcore and bridge is not in `~/.rainbowup`.

Reply:
This was semi implemented.